pkg/apis/operator/v1alpha1/zz_generated.openapi.go

```diff
@@ -109,7 +109,7 @@ func schema_pkg_apis_operator_v1alpha1_KubeFedWebHook(ref common.ReferenceCallba
 	return common.OpenAPIDefinition{
 		Schema: spec.Schema{
 			SchemaProps: spec.SchemaProps{
-				Description: "KubeFedWebHook is the Schema for the kubefedwebhooks API",
+				Description: "KubeFedWebHook is the Schema for the installs API",
```

Comment:
Should this description be changed since this is autogen code?

Reply:
Everything is auto-generated. I didn't change anything manually.